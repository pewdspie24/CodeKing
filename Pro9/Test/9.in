10
3126135835 4665969884
1225905354 2589266050
3766074022 7280720500
3723573919 6293295711
2086101042 3388476906
3408285057 6168614144
695851399 3919477094
3723450079 7410661711
1613778169 3336552728
1438296730 3219352893
