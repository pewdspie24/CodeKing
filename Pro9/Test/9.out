15993791
15246860
28254358
20979336
14781419
22471085
33941962
28877533
18730782
18743984
